docker-entrypoint.sh

```diff
@@ -27,15 +27,16 @@ fi
 ipwithnetmask="$(ip -f inet addr show dev eth0 | awk '/inet / { print $2 }')"
 ipaddress="${ipwithnetmask%/*}"
 
+[ -z "$ADVERTISED_HOSTNAME" ] && ADVERTISED_HOSTNAME="localhost"
```

Comment:
please default it to `ipaddress` so it's backward compatible.
(and you have this set explicitly in the compose file, which is good)